addons/hw_drivers/iot_handlers/drivers/DisplayDriver_L.py

```diff
@@ -63,13 +60,6 @@ def get_default_display(cls):
         displays = list(filter(lambda d: iot_devices[d].device_type == 'display', iot_devices))
         return len(displays) and iot_devices[displays[0]]
 
-    def run(self):
-        while self.device_identifier != 'distant_display' and not self._stopped.is_set():
-            time.sleep(60)
-            if self.url != 'http://localhost:8069/point_of_sale/display/' + self.device_identifier:
-                # Refresh the page every minute
-                self.call_xdotools('F5')
```

Comment:
Be careful when removing this line because it allowed you to update custom urls every 60 seconds. Urls not linked to Odoo display

https://github.com/odoo/enterprise/blob/17.0/iot/models/iot.py#L72

Reply:
@vlst-odoo There you go, that's the answer why it's there. Perhaps you also put as exception the url of the customer display so that it doesn't refreshed every 60 sec.

Reply:
ok! It's done

Reply:
And so you removed code without understanding what it did and without asking the team who made it what it was used for?
However, we are not far away.
Now have you tested all the flows linked to the screens?